packages/reactant-module/src/core/createStore.ts

```diff
@@ -153,7 +164,12 @@ export function createStore<T = any>({
       const services: IService[] = container.getAll(ServiceIdentifier);
       loadedModules.add(ServiceIdentifier);
       services.forEach((service, index) => {
-        handlePlugin(service, pluginHooks);
+        const indexPlugin = allServiceIdentifierKeys.indexOf(ServiceIdentifier);
+        if (indexPlugin === -1) {
+          pushPlugin(service, _pushPluginHooks, indexPlugin);
```

Comment:
`indexPlugin` seems not need

also, maybe the name should be `pluginIndex`

Reply:
sure, thx.